bitnami/apache/values.yaml

```diff
@@ -385,6 +385,34 @@ ingress:
   ##
   secrets: []
 
+## Apache Pod Disruption Budget configuration
+## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+## @param pdb.create Enable a Pod Disruption Budget creation
+## @param pdb.minAvailable Minimum number/percentage of pods that should remain scheduled
+## @param pdb.maxUnavailable Maximum number/percentage of pods that may be made unavailable
+##
+pdb:
+  create: false
+  minAvailable: 1
+  maxUnavailable: ""
+
+## Apache Autoscaling parameters
+## ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+## @param autoscaling.enabled Enable autoscaling for Apache deployment
+## @param autoscaling.apiVersion API Version of the HPA object (for compatibility with Openshift)
+## @param autoscaling.minReplicas Minimum number of replicas to scale back
+## @param autoscaling.maxReplicas Maximum number of replicas to scale out
+## @param autoscaling.targetCPU Target CPU utilization percentage
+## @param autoscaling.targetMemory Target Memory utilization percentage
+##
+autoscaling:
+  enabled: false
+  apiVersion: autoscaling/v2beta1
```

Comment:
This info should not come from a parameter. There are two options here:
- We hardcode the apiVersion to `autoscaling/v2beta1` in the manifest.
- We create a macro in the "common" chart that returns the appropriate version for HPA based on the Kubernetes version (see https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_capabilities.tpl)

Reply:
Okay, I copied that bit from the `bitnami/kong` chart because it was more flexible than hardcoding it, and I didn't want start fiddling with the `bitnami/common` chart.

If you're happy with hardcoding it, this is what I'll do (as it is currently done in most bitnami charts using autoscaling) :+1:

Reply:
Fair enough